README.md

```diff
@@ -55,7 +56,7 @@ Choose one of the following methods:
 $ niv add Mic92/sops-nix
```

Comment:
Is this still the current recommendation? [Mic92's personal configuration](https://github.com/Mic92/dotfiles/tree/master/nixos) seems to use flakes (see the `flake.nix` file at the root.)

Reply:
Maybe we can move flakes up and recommend `niv` for people not using flakes?